LCLSGeneral/LCLSGeneral/POUs/Hardware/FB_AnalogInput.TcPOU

```diff
@@ -15,9 +15,24 @@ VAR_INPUT
     fTermMax: LREAL;
     // The fReal value correlated with the terminal's min value
     fTermMin: LREAL;
+    // Value to scale the end result to
+    {attribute 'pytmc' := '
+        pv: RES
+        io: io
+    '}
+    fResolution : LREAL := 0.1;
+    {attribute 'pytmc' := '
+        pv: OFF
+        io: io
+    '}
+    fOffset : LREAL;
 END_VAR
 VAR_OUTPUT
     // The real value read from the output
+    {attribute 'pytmc' := '
+        pv: VAL
```

Comment:
Not sure if we have a standard PV for this.

Reply:
VAL is OK but it feels weird because it is usually used as a `.VAL` setpoint field

I don't have any immediate other suggestions, it probably works out pretty well and my misgivings are misplaced:
```
IM3L0:PPM:FWM:VAL_RBV
```

The above full PV looks pretty clear to me